data_model/derive/Cargo.toml

```diff
@@ -11,7 +11,7 @@ license.workspace = true
 proc-macro = true
 
 [dependencies]
-syn = "1.0.107"
+syn = { version = "1.0.107", features = ["full", "extra-traits"] }
```

Comment:
If this is used once, it should be used everywhere. We don't win anything by reducing the feature set, not even compilation time.

Reply:
this is tracked by #3289